common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -628,6 +635,10 @@ div.problem {
       height: ($baseline*2);
       vertical-align: middle;
       font-weight: 600;
+
+      @media print {
+        display: none;
```

Comment:
I can understand some of the other removed elements (borders, details, tangential navigation controls, etc) but at some point removing elements like the input buttons presumes some level of use / intent around the printed document. Is the expectation here that users printing these pages expect the problems to not reflect their input controls? It also seems a bit odd that checkbox controls would remain unchanged but multiple choice problems would have their buttons stripped. Thoughts? (@jolyonb)